opm/simulators/linalg/ParallelOverlappingILU0.hpp

```diff
@@ -747,13 +841,15 @@ class ParallelOverlappingILU0
 
         const size_type iEnd = lower_.rows();
         const size_type lastRow = iEnd - 1;
+        size_type interiorStart = iEnd - interiorSize_;
```

Comment:
Name is a bit misleading. `upperLoopStart` or `noGhosts`?

Reply:
Changed to upperLoppStart.